### DISCUSSION TITLE
My Company has disabled Microsoft store. How to install without Microsoft Store?

### DISCUSSION THREAD

**Discussion by farazzy4u** [tom-englert/RegionToShare#20] [Q&A] [answered]
My company has disabled Microsoft Store, because of this I am not able to install it. Can you tell me a workaround where I can install Region-To-Share without Microsoft Store into my company's laptop ?

**Comment by tom-englert** [comment, marked as answer]
https://github.com/tom-englert/RegionToShare#installation

**Reply by eschoolcraft** [reply]
This doesn't work on my setup.  I installed via the github and each time I run it there is a long pause of like 30 seconds and eventually it will say "you need to install an app for this task" and wants me to load the microsoft store, which I cannot do because it has been disabled by the IT group.  If I say "No" when it eventually comes up, it runs RegionToShare, but the wait is unfeasible.  Got a solution to the "you need an app" issue?
